#request
string inputString
---
#response
string modifiedString
